src/components/SpruceForm/SpruceForm.test.tsx

```diff
@@ -124,3 +324,46 @@ const basicForm = {
     },
   },
 };
+
+const textInput = (emptyValue?: string) => ({
+  formData: {},
+  schema: {
+    type: "object" as "object",
+    properties: {
+      textInput: {
+        type: "string" as "string",
+        title: "Text Input",
+        default: "",
+        minLength: 1,
+      },
+    },
+  },
+  uiSchema: {
+    textInput: {
+      "ui:data-cy": "text-input",
+      ...(emptyValue && { "ui:emptyValue": emptyValue }),
+    },
+  },
+});
+
+const textArea = (emptyValue?: string) => ({
+  formData: {},
+  schema: {
+    type: "object" as "object",
+    properties: {
+      textArea: {
+        type: "string" as "string",
+        title: "Text Area",
+        default: "",
+        minLength: 1,
+      },
+    },
+  },
+  uiSchema: {
+    textArea: {
+      "ui:widget": widgets.TextareaWidget,
```

Comment:
[nit] You can simply refer to this as `"textarea"` like we do [here](https://github.com/evergreen-ci/spruce/blob/6c09c39220d1a3d8444d8782696e5d5cba647be3/src/components/SpruceForm/SpruceForm.stories.tsx#L99) because we override the default widgets [here](https://github.com/evergreen-ci/spruce/blob/6c09c39220d1a3d8444d8782696e5d5cba647be3/src/components/SpruceForm/Widgets/index.tsx#L15) (there are other places in the codebase where this change could be made too)
```suggestion
      "ui:widget": "textarea",
```